### PR TITLE
Add support for Avskrivning (Sign Off)

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

**Comment by douzounov** [comment]
> @teodoradobreva , @douzounov , do you think I should also update the models?
> 
> If yes, we have to merge PR#13 first (as it edits the models too and resolving conflicts in PNGs would not be fun).

It's not so important to update the models now. We can do it later.

File: src/02 - Object types.md [hunk 1]
```diff
@@ -333,6 +333,8 @@ The following notation is used in the tables below:
...
 |      | ref             | refNoekkelord                 |          |             |           |  |      |      |   x   |      |       x       | <p>&bull; *unlink* not supported in Documaster</p>                                                                                                                                                                                                                                                     | &nbsp;                                                                                                                                                 |
+|      | ref             | refAvskrivning                |          |             |           |  |      |      |   x   |      |       x       |                                                                                                                                                                                                                                                                                                        | &nbsp;                                                                                                                                                 |
```
Comment (teodoradobreva) [inline src/02 - Object types.md]: Unlinking `refAvskrivning`/`refTilknyttetAvskrivning` is not supported in Documaster.

File: src/02 - Object types.md [hunk 2]
```diff
@@ -362,6 +364,22 @@ The following notation is used in the tables below:
...
+| M601 | string | opprettetAvBrukerIdent   |    x     |      x      |           |  |  cw  |  x   |   x   |  x   |               |                                                                                             |           |
+| M619 | string | avskrivningsmaate        |    x     |             |     x     |  |  x   |  x   |   x   |  x   |               | <p>&bull; code list in Documaster</p>                                                       |           |
+|      |        |                          |          |             |           |  |      |      |       |      |               |                                                                                             |           |
+|      | ref    | refJournalpost           |    x     |             |           |  |      |  x   |   x   |      |       x       | may be partially supported by systems</p><p>&bull; *unlink* not supported in Documaster</p> |           |
+|      | ref    | refTilknyttetJournalpost |          |             |           |  |      |      |   x   |      |       x       | may be partially supported by systems</p><p>&bull;                                          | &nbsp;    |
```
Comment (teodoradobreva) [inline src/02 - Object types.md]: Unlinking `refTilknyttetJournalpost` is also not supported in Documaster.

Formatting of `Comment 1` for both refs is broken (though parsable); there are missing opening/closing paragraph tags.

File: src/02 - Object types.md (hunk 1 above)
Comment (ivaylomitrev) [reply src/02 - Object types.md]: I partially agree. `refAvskrivning` cannot be unlinked, but `refTilknyttetAvskrivning` can. I've modified it accordingly.

File: src/02 - Object types.md (hunk 2 above)
Comment (ivaylomitrev) [reply src/02 - Object types.md]: As per my previous comment - linking/unlinking `refTilknyttetJournalpost` is supported.

I have fixed the incomplete paragraph tags though.

**Comment by ivaylomitrev** [comment]
Squashed and rebased in prepration for merge.

**Pull request merged**